code/game/machinery/computer/abnormality_work.dm

```diff
@@ -111,7 +122,7 @@
 		var/datum/suppression/information/I = GetCoreSuppression(/datum/suppression/information)
 		if(!tutorial && istype(I))
 			work_display = Gibberish(work_display, TRUE, I.gibberish_value)
-		if(HAS_TRAIT(user, TRAIT_WORK_KNOWLEDGE))
+		if(HAS_TRAIT(user, TRAIT_WORK_KNOWLEDGE) || (!mechanical_upgrades["workrate"] || mechanical_upgrades["workrate"] != 0))
```

Comment:
```suggestion
		if(HAS_TRAIT(user, TRAIT_WORK_KNOWLEDGE) || (mechanical_upgrades["workrate"] || mechanical_upgrades["workrate"] != 0))
```

Reply:
Yeah that was a major accident on my part.